e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -0,0 +1,477 @@
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "IOTHUB-PROXY-SERVER-ADDRESS" --value $proxyServerAddress --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "FAR-AWAY-IOTHUB-HOSTNAME" --value "$FarHubName.azure-devices.net" --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "DPS-IDSCOPE" --value $dpsIdScope --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "PROVISIONING-CONNECTION-STRING" --value $deviceProvisioningServiceConnectionString --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "CUSTOM-ALLOCATION-POLICY-WEBHOOK" --value "https://drwe2efnapp.azurewebsites.net/api/HttpTrigger1?code=PtayXB9/P7E5Wt0AmmSs5oO1n0n3jNEM3zDkq3U8VQSziNSMCINOGw==" --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "DPS-GLOBALDEVICEENDPOINT" --value "global.azure-devices-provisioning.net" --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "DPS-X509-PFX-CERTIFICATE-PASSWORD" --value $DPS_X509_PFX_CERTIFICATE_PASSWORD --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "IOTHUB-X509-PFX-CERTIFICATE" --value $IOTHUB_X509_PFX_CERTIFICATE --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "DPS-INDIVIDUALX509-PFX-CERTIFICATE" --value $DPS_INDIVIDUALX509_PFX_CERTIFICATE --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "DPS-GROUPX509-PFX-CERTIFICATE" --value $DPS_GROUPX509_PFX_CERTIFICATE --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "DPS-GROUPX509-CERTIFICATE-CHAIN" --value $DPS_GROUPX509_CERTIFICATE_CHAIN --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "IOTHUB-DEVICE-CONN-STRING-INVALIDCERT" --value "HostName=invalidcertiothub1.westus.cloudapp.azure.com;DeviceId=DoNotDelete1;SharedAccessKey=zWmeTGWmjcgDG1dpuSCVjc5ZY4TqVnKso5+g1wt/K3E=" --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "IOTHUB-CONN-STRING-INVALIDCERT" --value "HostName=invalidcertiothub1.westus.cloudapp.azure.com;SharedAccessKeyName=iothubowner;SharedAccessKey=Fk1H0asPeeAwlRkUMTybJasksTYTd13cgI7SsteB05U=" --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "PROVISIONING-CONNECTION-STRING-INVALIDCERT" --value "HostName=invalidcertdps1.westus.cloudapp.azure.com;SharedAccessKeyName=provisioningserviceowner;SharedAccessKey=lGO7OlXNhXlFyYV1rh9F/lUCQC1Owuh5f/1P0I1AFSY=" --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "STORAGE-ACCOUNT-CONNECTION-STRING" --value $storageAccountConnectionString --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "LA-WORKSPACE-ID" --value $workspaceId --output none
+az keyvault secret set --vault-name $SecretsKeyVaultName --name "LA-AAD-TENANT" --value "72f988bf-86f1-41af-91ab-2d7cd011db47" --output none
```

Comment:
can this be derived from `az account show`?

Reply:
What do you mean? We can do the other way which is write to KV but don't ready to set the Values, We can read from KV only on subsequent runs but as there was code to already read it, I just reused that.

Reply:
I'm saying we're hard coding this to MS tenant. The identity authenticated may not be MS.

But we did ask them to log in with `az account login`, and we capture info about them from that call. Is the tenant Id available from that call?

The tenant of the app Id is going to be the same as their login, right?